sdk/communication/communication-phone-numbers/test/public/siprouting/getRoutes.spec.ts

```diff
@@ -55,13 +55,13 @@ matrix([[true, false]], async function (useAad) {
     it("can retrieve not empty routes", async () => {
       const expectedRoutes = [
         {
-          name: "myFirstRoute",
+          name: "Sanitized",
```

Comment:
perhaps we should skip `AZSDK3493` here as well since there's no reason to sanitize this property

Reply:
I tried to use AZSDK3493 but it didn't work on this 
https://github.com/Azure/azure-sdk-for-js/pull/28607/files#diff-077893faed356a6fafc16f514eeb779c08e4dc6ce8beeacd904e03a40d023215R71

Reply:
updated, it should have been resolved.